### PR TITLE
update formatting of cli windows.md

### PR DESCRIPTION
**Pull request by etseidler**
in my opinion, this makes the page more readable.

### CODE REVIEW

File: x/docs/md/cli/windows.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 ## Windows
 
-Below are instructions for install using the most common methods - using Powershell & Chocolately Package Manager. For further help and instructions, see:
+Below are instructions for install using the most common methods - using Powershell & Chocolately Package Manager. For further help and instructions, see the following:
+
 1. [Install Alternatives for instructions on installing manually on windows](/cli/install)
-2. [exercism.io general help](http://exercism.io/help)
-3. [join the exercism.io chat on gitter](https://gitter.im/exercism/support): [![Join the chat at https://gitter.im/exercism/support](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/exercism/support)
+1. [exercism.io general help](http://exercism.io/help)
+1. [join the exercism.io chat on gitter](https://gitter.im/exercism/support): [![Join the chat at https://gitter.im/exercism/support](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/exercism/support)
```
Comment (kytrinyx) [inline x/docs/md/cli/windows.md]: Markdown automatically increments the numbers. By leaving them as the same number it makes it easier to reorder lists.

Comment (kytrinyx) [inline x/docs/md/cli/windows.md]: Ah, I also realized that you might be looking at this in a text editor. This documentation is rendered on the site here: http://exercism.io/cli/windows

**Comment by kytrinyx** [comment]
@etseidler Will you have time to make these changes or should I go ahead and make them for you?

**Comment by etseidler** [comment]
Go for it.

-- sent from my phone

**Comment by kytrinyx** [comment]
Merged in 5eb67ee92a44fd61e928cf3aece43aeb743c4fe2

**Pull request closed without merging**